lib/web_ui/test/engine/platform_dispatcher_test.dart

```diff
@@ -160,6 +162,34 @@ void testMain() {
       expect(ui.PlatformDispatcher.instance.textScaleFactor, findBrowserTextScaleFactor());
     });
   });
+
+  test('ViewConfiguration asserts that both window and view are not provided', () {
+    expect(() {
+      // ignore: deprecated_member_use
+      final EngineFlutterWindow window = EngineFlutterWindow(0, ui.PlatformDispatcher.instance);
+      ui.ViewConfiguration(
+        window: window,
+        view: window,
+      );
+    }, throwsAssertionError);
+  });
+
+  test("A ViewConfiguration's view and window are backed with the same property", () {
+    final ui.FlutterView window = EngineFlutterWindow(0, ui.PlatformDispatcher.instance);
+    final ui.ViewConfiguration viewConfiguration = ui.ViewConfiguration(view: window);
+    // ignore: deprecated_member_use
+    expect(viewConfiguration.window, window);
+    // ignore: deprecated_member_use
+    expect(viewConfiguration.window, viewConfiguration.view);
```

Comment:
Can you also make sure that the `window` argument works? Just copy the test but initialize with `window`.

Reply:
https://github.com/flutter/engine/pull/38453/commits/6d531ba35e27e11034e5ed3ee44d283dae81b3e0 should resolve this.